eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -55,23 +55,26 @@ parameters:
     type: object
     default:
       Public:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
         ServiceConnection: azure-sdk-tests
         SubscriptionConfigurationFilePaths:
           - eng/common/TestResources/sub-config/AzurePublicMsft.json
       Preview:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
         ServiceConnection: azure-sdk-tests-preview
+        SubscriptionConfigurationFilePaths:
```

Comment:
I believe we should only make this switch after a language as enabled UseFederatedAuth as the default otherwise anyone not using it will be broken unless the override their own configs.